docs/config/keys.md

```diff
@@ -20,53 +26,163 @@ return {
+ * `SHIFT` - The shift key, left and right variants are equivalent
+ * `ALT`, `OPT`, `META` may be used for platform agnostic configs & are all equivalent to: 
+   * on macOS the `Option` key
+   * on other platforms the `Alt` or `Meta` key, left and right variants are equivalent
+ * `LEADER` - a special modal modifier state managed by `wezterm`.
+    See [Leader Key](#leader-key) for more information.
  * `VoidSymbol` - This keycode is emitted in special cases where the original
    function of the key has been removed. Such as in Linux and using `setxkbmap`.
    `setxkbmap -option caps:none`. The `CapsLock` will no longer function as
    before in all applications, instead emitting `VoidSymbol`.
 
-You can combine modifiers using the `|` symbol (eg: `"CMD|CTRL"`).
+You can combine modifiers using the `'|'` symbol e.g.
 
-The `key` value can be one of the following keycode identifiers.  Note that not
-all of these are meaningful on all platforms:
-
-`Hyper`, `Super`, `Meta`, `Cancel`, `Backspace`, `Tab`, `Clear`, `Enter`,
-`Shift`, `Escape`, `LeftShift`, `RightShift`, `Control`, `LeftControl`,
-`RightControl`, `Alt`, `LeftAlt`, `RightAlt`, `Menu`, `LeftMenu`, `RightMenu`,
-`Pause`, `CapsLock`, `VoidSymbol`, `PageUp`, `PageDown`, `End`, `Home`, `LeftArrow`,
-`RightArrow`, `UpArrow`, `DownArrow`, `Select`, `Print`, `Execute`,
-`PrintScreen`, `Insert`, `Delete`, `Help`, `LeftWindows`, `RightWindows`,
-`Applications`, `Sleep`, `Numpad0`, `Numpad1`, `Numpad2`, `Numpad3`,
-`Numpad4`, `Numpad5`, `Numpad6`, `Numpad7`, `Numpad8`, `Numpad9`, `Multiply`,
-`Add`, `Separator`, `Subtract`, `Decimal`, `Divide`, `NumLock`, `ScrollLock`,
-`BrowserBack`, `BrowserForward`, `BrowserRefresh`, `BrowserStop`,
-`BrowserSearch`, `BrowserFavorites`, `BrowserHome`, `VolumeMute`,
-`VolumeDown`, `VolumeUp`, `MediaNextTrack`, `MediaPrevTrack`, `MediaStop`,
-`MediaPlayPause`, `ApplicationLeftArrow`, `ApplicationRightArrow`,
-`ApplicationUpArrow`, `ApplicationDownArrow`, `F1`, `F2`, `F3`, `F4`,
-`F5`, `F6`, `F7`, `F8`, `F9`, `F10`, `F11`, `F12`, `F13`, `F14`, `F15`,
-`F16`, `F17`, `F18`, `F19`, `F20`, `F21`, `F22`, `F23`, `F24`.
+```lua
+local wezterm = require 'wezterm'
+return {
+  keys = {
+    -- Key assignment using a multi-key modifier
+    {
+      key = 'R',
+      mods = 'SHIFT|SUPER',
+      action = wezterm.action.ReloadConfiguration,
+    },
+  },
+}
+```
 
 Alternatively, a single unicode character can be specified to indicate
 pressing the corresponding key.
 
-Pay attention to the case of the text that you use and the state of the `SHIFT` modifier, as `key="A"` will match 
+Pay attention to the case of the text that you assign to `key = '<some key>'` if your modifier contains `SHIFT` as this is a common mistake.<br>
+As of *20221119-145034-49b9839f* there's an open issue on this, see [#1906](https://github.com/wez/wezterm/issues/1906).
+```lua
+local wezterm = require 'wezterm'
+return {
+  keys = {
+    -- This will not be abled to be triggered
```

Comment:
@wez I'm able to trigger keybinding dusch as `{ key = "h", mods = "ALT|SHIFT", ... }`, is this documentation only half true?

Reply:
did you check out #1906 ?

Reply:
i think you're right, this should be bette clarified as to what the issue actually is. I believe this should be given a better example since to my knowledge it issue only effects non alphabetic characters

Reply:
this should provide clarity. I had to do a sanity check on it myself.